build/mediaelementplayer.css

```diff
@@ -51,7 +51,8 @@ Reference: http://blog.rrwd.nl/2015/04/04/the-screen-reader-text-class-why-and-h
 }
 
 .mejs__container:focus {
-    outline: none;
+    outline-offset: 0.125rem;
```

Comment:
No build files should be added with a feature branch, because it creates merge conflicts. We only build the files, when we add a new version.